pkg/klusterlet/nodecollector/nodecollecter.go

```diff
@@ -108,6 +109,10 @@ func (r *resourceCollector) reconcile(ctx context.Context) {
 		klog.Errorf("Failed to get cluster: %v", err)
 	}
 
+	if utils.ClusterIsOffLine(clusterinfo.Status.Conditions) {
```

Comment:
I do not think this is needed. Are you  saying cluster is offline but addon is  still running?

Reply:
yes, add this for this case.